src/containers/Main.tsx

```diff
@@ -84,6 +90,7 @@ const Main = () => {
           appId={process.env.REACT_APP_PRIVY_APP_ID as string}
           config={{
             appearance: { theme: 'dark' },
+            defaultChain: process.env.REACT_APP_USE_TESTNETS === 'true' ? goerli : mainnet,
```

Comment:
Given that Goerli is sunset should we use this opportunity to move to Sepolia? Will need to check actually if Prime SDK supports this

Reply:
Yes it's supported: https://sdk.etherspot.io/enums/networknames#sepolia

Reply:
makes sense, changed